packages/core/src/actions/sendTransaction.ts

```diff
@@ -74,7 +75,15 @@ export async function sendTransaction<
   else
     client = await getConnectorClient(config, { account, chainId, connector })
 
+  const { connector: activeConnector } = getAccount(config)
+
   const gas = await (async () => {
+    // Skip gas estimation if `data` doesn't exist (likely not a contract interaction).
+    if (!('data' in parameters) || !parameters.data) return undefined
```

Comment:
We should skip estimating gas if we know that the transaction is not a contract interaction as all wallets will disallow value transfer if the user does not have enough funds.